homeassistant/components/modem_callerid/config_flow.py

```diff
@@ -32,10 +32,10 @@ def __init__(self) -> None:
 
     async def async_step_usb(self, discovery_info: usb.UsbServiceInfo) -> FlowResult:
         """Handle USB Discovery."""
-        device = discovery_info["device"]
+        device = discovery_info[usb.ATTR_DEVICE]
 
         dev_path = await self.hass.async_add_executor_job(usb.get_serial_by_id, device)
-        unique_id = f"{discovery_info['vid']}:{discovery_info['pid']}_{discovery_info['serial_number']}_{discovery_info['manufacturer']}_{discovery_info['description']}"
+        unique_id = f"{discovery_info[usb.ATTR_VID]}:{discovery_info[usb.ATTR_PID]}_{discovery_info[usb.ATTR_SERIAL_NUMBER]}_{discovery_info[usb.ATTR_MANUFACTURER]}_{discovery_info[usb.ATTR_DESCRIPTION]}"
         if (
             await self.validate_device_errors(dev_path=dev_path, unique_id=unique_id)
             is None
```

Comment:
Sorry I wasn't clear in the last PR.  The goal was to separate the usb integration changes from other integrations so we can point to the USB PR as an example in the future for other contributors to use the new constants